prog/hints_test.go

```diff
@@ -313,6 +364,7 @@ func TestHintsShrinkExpand(t *testing.T) {
 			// int64, so we model this accordingly.
 			"Shrink with a wider replacer test2",
 			0x1234,
+			2,
```

Comment:
Could we do `if inSize == 0 { inSize = 8 }` in the execution engine instead of all these explicit sizes? That would reduce the diff and also relief people adding tests that not testing size handling from figuring out what is the right value.

Reply:
If I remove these sizes then I have to add the name of the fields when I create a new `ConstArgTest` object. Do we want this change?

Reply:
Good point. I missed that.
Then let's leave it as you have it now.